case-studies/simple/draft1.md

```diff
@@ -0,0 +1,215 @@
+
+The hypertension patients needed friends. Someone to coax them back into the clinic. "This is important, remember? We want you to come here so you stay healthy." There were automated text message reminders. In India, this is already extremely common so there was little surprise to patients. An Indian broadband company will text you to remind you to pay your bill. An Indian bank will drop you an SMS whenever there's a transaction on your account. This was hardly any different. Even more convincing than an SMS was a phone call from a real human being. This left the nilenso team to come up with a phone masking technique, similar to those used in taxi-hailing apps like Uber. If your cab driver doesn't want you to know her personal phone number, your nurse _definitely_ doesn't want you knowing hers. It doesn't take much imagination to see how a nurse's number out in the wild quickly transforms them into a 24/7 personal healthcare service.
+
+Akshay Gupta, who had recently joined his nilenso colleagues on the Simple team at this stage, describes the phone masking system: "It's a bit fun, really. Because the entire system has to assume 'offline-first' in every scenario, phone masks can't rely on anything digital. The way we got around that was to provide the nurse with a way to call the patient's number through a central number. The patient's number is actually encoded in that initial phone call using DTMF codes and that call is then routed accordingly to the patient from a number which is not the nurse's." This old-school hack was somewhat reminiscent of the phone phreaking of the 1970s, a technique that perhaps would have amused the original phone hackers like John "Captain Crunch" Draper. [^john-draper]
```

Comment:
Haha, so it is clever and everything, but it might be valuable to caveat that this system only has a 65% success rate because of bad infstracture and telecom networks are unreliable and the potentially longer-term solution to this problem is to have a call center.

Reply:
:woman_shrugging: We can definitely do that, but remember it's not a white paper. I've elided a lot of details across the entire story due to either my own ignorance or actively disregarding them because they don't flow. Of course, if at any point the story is painting nilenso in a light which is actively dishonest, we should make sure we address that. It's your call whether this is one of those cases.